### PR TITLE
[11.0][ADD] purchase_stock_price_unit_sync: Update cost price in stock…

### PR DESCRIPTION
**Pull request by carlosdauden**
… moves already done

@Tecnativa  TT19351

### CODE REVIEW

**Review by rafaelbn — APPROVED** [review]
:+1:

File: purchase_stock_price_unit_sync/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This module allows to sync stock moves price unit with purchase order line
+price when moves already done.
```
Comment (pedrobaeza) [inline purchase_stock_price_unit_sync/readme/DESCRIPTION.rst]:
```suggestion
price when moves are already done.
```
but the update is performed no matter the stock move is done or not

File: purchase_stock_price_unit_sync/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+This module allows to sync stock moves price unit with purchase order line
```
Comment (pedrobaeza) [inline purchase_stock_price_unit_sync/readme/DESCRIPTION.rst]: This is user documentation, so we should adapt the terminology

```suggestion
This module allows to sync picking cost prices with purchase order line
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Travis fails due to maximum recursion... maybe forgot a context??

**Comment by carlosdauden** [comment]
Travis green

**Comment by pedrobaeza** [comment]
You can migrate it to 12.0 now

**Pull request merged**